frontend/web/components/ButterBar.tsx

```diff
@@ -18,9 +19,12 @@ const ButterBar: React.FC<ButterBarProps> = ({ billingStatus, projectId }) => {
   const matches = document.location.href.match(/\/environment\/([^/]*)/)
   const environment = matches && matches[1]
   const timerRef = useRef<NodeJS.Timer>()
-  const { data: featureImports, refetch } = useGetFeatureImportsQuery({
-    projectId,
-  })
+  const { data: featureImports, refetch } = useGetFeatureImportsQuery(
+    {
+      projectId,
+    },
+    { skip: !projectId },
```

Comment:
Fixes an observed issue where we were sending requests to a url with project as undefined